Makefile

```diff
@@ -0,0 +1,21 @@
+# virtual-kubelet-saladcloud
+
+.PHONY: build
+build: CGO_ENABLED=0
+build:
+	go build -o ./bin/vksc ./cmd/virtual-kubelet/main.go
```

Comment:
```suggestion
	go build -o ./bin/virtual-kubelet ./cmd/virtual-kubelet/main.go
```

Reply:
Wasn't sure what the name should be; do they all use virtual-kubelet as the binary name?  I imagine while troubleshooting you would know which one it is by the command args.

Reply:
It is a bit odd at first, but the convention I've seen applied by others is to name the binary `virtual-kubelet`. The upstream Virtual Kubelet project is a _library_. The executable is _always_ baked against a specific provider. You'll know which one it is- you had to get the Helm chart or OCI image and deploy it.